CamTool/Assets/Scripts/cameraPicture.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class cameraPicture : MonoBehaviour
+{
+    [Tooltip("Set button for taking screenshots")]
+    public Button takePictureButton;
+    /// <summary>
+    /// Change path for screenshots
+    /// </summary>
+    string path = @"C:/UnityScreenshots/";
```

Comment:
Make this public or at least serialized, fields without access modifiers default to private.